.circleci/config.yml

```diff
@@ -32,7 +32,3 @@ jobs:
         
       # run tests!
       - run: mvn integration-test
-
-      - store_test_results:
-          path: target/surefire-reports
```

Comment:
Why do you remove this? This way circleci won't read and display test results.

Reply:
Merging error. Will reintroduce it.